### PR TITLE
[NVIDIA] int64 support

### PR DESCRIPTION
**Pull request by apavliuk-altran**
### Details:
- *added support of `int64` data type*
### Ticket:
- 102680
- 102832

### CODE REVIEW

File: modules/nvidia_plugin/src/cuda_plugin.cpp [hunk 1]
```diff
@@ -39,39 +39,50 @@ InferenceEngine::IExecutableNetworkInternal::Ptr Plugin::LoadExeNetworkImpl(cons
...
+        return supportedPrecisions.count(precision) != 0;
+    };
+
+    auto getSupportedNames = [&supportedPrecisions] {
+        std::stringstream ss;
+        for (const auto& p : supportedPrecisions) {
+            ss << p.name() << ' ';
+        }
+        return ss.str();
+    };
```
[the remark this answers is not in the document]
Comment (nkogteva) [reply modules/nvidia_plugin/src/cuda_plugin.cpp:67]: As I already said previously: we should not have such limitations in this place. All precision limitations should be determined by precisions supported by primitives.

**Review by nkogteva — CHANGES_REQUESTED** [review]

**Comment by nkogteva** [comment]
Also would be nice to enable some tests for i64

File: modules/nvidia_plugin/src/cuda_plugin.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ytorzuk-altran) [reply modules/nvidia_plugin/src/cuda_plugin.cpp:67]: @ilya-lavrenov @nkogteva There is a check of supported `User Precision` to `Network Precision` converters implemented in [NVIDIA Plugin](https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/nvidia_plugin/src/cuda_infer_request.cpp#L649) Implementation of the NVIDIA plugin implies model inputs conversion by means of CPU during preprocessing step of model execution pipeline.

[the remark this answers is not in the document]
Comment (nkogteva) [reply modules/nvidia_plugin/src/cuda_plugin.cpp:67]: > I didn't get how it is connected with this one check. This check is applied in all cases even pre-processing is not needed. Is not it?

[the remark this answers is not in the document]
Comment (apavliuk-altran) [reply modules/nvidia_plugin/src/cuda_plugin.cpp]: Removed

**Comment by apavliuk-altran** [comment]
> Also would be nice to enable some tests for i64

We've added the tests.

File: modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp [hunk 2]
```diff
@@ -27,10 +27,12 @@ const std::vector<std::vector<size_t>> smoke_numSplits = {
     {4, 11, 6, 9},
 };
 
+const std::vector<size_t> smoke_axis = {0, 1, 2, 3};
+
 INSTANTIATE_TEST_CASE_P(smoke_NumSplitsCheck,
                         VariadicSplitLayerTest,
                         ::testing::Combine(::testing::ValuesIn(smoke_numSplits),
-                                           ::testing::Values(-3, -2, -1, 0, 1, 2, 3),
```
Comment (nkogteva) [inline modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: Why did you removed negative values?

**Review by nkogteva — APPROVED** [review]

**Comment by nkogteva** [comment]
In general LGTM

File: modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp (hunk 2 above)
Comment (apavliuk-altran) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: Axis type in `VariadicSplitParams` is or `size_t`
https://github.com/openvinotoolkit/openvino/blob/288a750bc6f370b9b332cd118d14feb21657805c/src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/variadic_split.hpp#L19

Comment (nkogteva) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: > Why did you remove negative values?

It is have to be fixed in test. VariadicSplit supports negative axis https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/VariadicSplit_1.md
```
2: axis. Axis along data to split. A scalar or tensor with shape [1] of type T2 with value from range -rank(data) .. rank(data)-1. Negative values address dimensions from the end. Required.
```

Comment (apavliuk-altran) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: The link I provided is the general test class in the main Openvino repository.
Do you propose that we should fix it there?

Comment (nkogteva) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: > The link I provided is the general test class in the main Openvino repository. Do you propose that we should fix it there?

Could you please check latest master? https://github.com/openvinotoolkit/openvino/pull/15925 was merged

Comment (apavliuk-altran) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp:33]: Updated it, thanks!

**Pull request merged**